crates/uv-resolver/src/prerelease_mode.rs

```diff
@@ -60,12 +60,15 @@ impl PreReleaseStrategy {
             PreReleaseMode::Allow => Self::Allow,
             PreReleaseMode::IfNecessary => Self::IfNecessary,
             PreReleaseMode::Explicit => Self::Explicit(
+                // TODO(charlie): Should requirements be included here, if they're overridden by
+                // overrides?
```

Comment:
Genuinely not sure how constraints and overrides should work in all these different TODOs.